ng2-components/ng2-alfresco-core/src/services/alfresco-api.service.ts

```diff
@@ -54,7 +54,7 @@ export class AlfrescoApiService {
     }
 
     private init() {
-        this.alfrescoApi = <AlfrescoApi>new alfrescoApi({
+        this.alfrescoApi = <AlfrescoApi>new AlfrescoApi({
```

Comment:
This should not work at runtime, tried that before and application crashes on start. Does it work for you?